src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -656,6 +657,15 @@ int sd_dhcp_client_set_fallback_lease_lifetime(sd_dhcp_client *client, uint64_t
         return 0;
 }
 
+int sd_dhcp_client_set_bootp(sd_dhcp_client *client, bool bootp) {
```

Comment:
Please use `int` rather than `bool` for functions prefixed with `sd_` even if it is a boolean.